assets/src/theme/VarPicker.js

```diff
@@ -149,8 +162,15 @@ export const VarPicker = (props) => {
       <input type="checkbox" readOnly checked={ shouldGroup }/>
       { 'Group last clicked element' }
     </label> }
-    { !collapsed && !!serverThemes && <ul style={{maxHeight: '140px'}}>
+    { !collapsed && serverThemesLoading && <span>Loading server themes...</span>}
```

Comment:
There are some small spacing issues here and on the following lines 😬 For consistency I think there should be no space after `{` and before `}`